chain/network/src/network_protocol/mod.rs

```diff
@@ -256,10 +255,6 @@ pub enum PeerMessage {
     /// Gracefully disconnect from other peer.
     Disconnect,
     Challenge(Challenge),
-    EpochSyncRequest(EpochId),
-    EpochSyncResponse(Box<EpochSyncResponse>),
-    EpochSyncFinalizationRequest(EpochId),
-    EpochSyncFinalizationResponse(Box<EpochSyncFinalizationResponse>),
```

Comment:
ouch, good catch! So we actually have the capability to send these messages over the network, they are succesfully decoded, and than just get discarded as unxepected messages.